examples/azureiot_esp32spi/azureiot_central_commands.py

```diff
@@ -103,9 +103,15 @@
 
 # pylint: enable=wrong-import-position
 
+pool = adafruit_connection_manager.get_radio_socketpool(esp)
+ssl_context = adafruit_connection_manager.get_radio_ssl_context(esp)
 # Create an IoT Hub device client and connect
 device = IoTCentralDevice(
-    socket, esp, secrets["id_scope"], secrets["device_id"], secrets["device_sas_key"]
+    pool,
+    ssl_context,
+    secrets["id_scope"],
+    secrets["device_id"],
+    secrets["device_sas_key"],
 )
```

Comment:
These changes are common for esp32spi:
1. get `pool` and `ssl_context` via `adafruit_connection_manager`
2. Pass to azure method

Reply:
Okay! Makes sense to me.